src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class 'calendar' is repeated. Consider removing the duplicate class to avoid redundancy.